src/apps/dashboard/routes/users/parentalcontrol.tsx

```diff
@@ -70,6 +70,13 @@ const UserParentalControl = () => {
     const [ blockedTags, setBlockedTags ] = useState<string[]>([]);
     const libraryMenu = useMemo(async () => ((await import('../../../../scripts/libraryMenu')).default), []);
 
+    // The following are meant to be reset on each render.
+    // These are to prevent multiple callbacks to be added to a single element in one render as useEffect may be executed multiple times in each render.
```

Comment:
If I'm correct, this is only specific to strict mode and won't happen in production?

Reply:
We are using Strict Mode, and in React 18 this will be executed twice.

Reply:
I think it would be cleaner to unbind the event listeners when useEffect is unmounted? Unless there is something barring this from happening that I missed.

Reply:
Makes sense. Now unbind all event listeners on unmounting